.changeset/happy-ears-call.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/react': minor
```

Comment:
Should this be a `patch` instead?

Reply:
I wasn't sure, so I went probably overly cautious. I couldn't really decide if this counted as a bug fix or additional functionality. I suppose it makes something work that didn't work before, as opposed to exposing new possibilities or options, so a patch is more appropriate? 
I'll update this. _Somehow_. 😅